OSSpecific.xml

```diff
@@ -3,9 +3,10 @@
 <project name="OSSpecific" default="dist" basedir=".">
 
   <!-- set global properties for this build -->
-  <property name="src" value="OSSpecific/src" />
-  <property name="build" value="OSSpecific/build" />
-  <property name="docs" value="OSSpecific/docs" />
+  <property name="src" value="${ant.project.name}/src" />
```

Comment:
These changes do not belong in this PR. If you would like to make some style changes to the build scripts as a whole, please raise an issue for discussion. This has nothing to do with removing the reflection code from `OSSpecific`.